src/client/components/SingleAnswerComponent/SingleAnswer.styles.css

```diff
@@ -0,0 +1,68 @@
+.single-answer-component h2 {
+  font-family: RammettoOne;
+  color: rgb(94, 36, 18);
+}
+
+.answer label {
+  font-weight: bolder;
+  margin-left: 2em;
+  font-size: 20px;
+}
+
+/* Customize the label (the container) */
+.answer-container {
+  border-radius: 5px;
+  width: 300px;
```

Comment:
A lot of large pixel based values for size and layout properties. Try to attain flexibility and responsibility in your designs.

Reply:
Hey @senner007. Do you mean like i should use max-width and min-width or should i change it to % instead of pixels? Or is it something else i need to do?

Reply:
Something along the lines of that yes. It ususally depends on the context. How does it look when you resize the screen?

Reply:
https://github.com/HackYourFuture-CPH/fp-class12/pull/29

Reply:
Please go through comments by @bytewiz 
https://github.com/HackYourFuture-CPH/fp-class12/pull/29